Week3 Assignment

yaditya20 opened the pull request:
This is the 1st Pull Request and below are the task that has been accomplished so far:

- [x] Portrait Layout of the app

- [x] Change Favorite Book on Click of a Button

- [x] Animation on Button Click and Book Change TextView

- [x] Landscape Layout of the app

- [x] Improve / Polish Layout

yaditya20 committed:
Initial Commit

yaditya20 committed:
Added Bounce and Blink Animation on Button and TextView resp.

yaditya20 committed:
Updated the App Launcher Icon

abunur commented:
I'm grading you based on your last commit before the homework deadline, which was 1 pm EST on Monday.

Your layout is not correct. You have a button that is overlapping with other UI elements in the view.

![screenshot-1592697572059](https://user-images.githubusercontent.com/4803881/85213751-b08c2c80-b330-11ea-965f-c913e61bc5ad.png)

You need to pay attention to the compiler warnings in Android Studio. You have four warnings just in MainActivity.kt. I would not accept this PR if it were a production app.

Even in the design view you can see your layout is not correct. It doesn't seem like you spent enough time or effort on this assignment.

Your grade for this assignment is "room for improvement". Please try to check your work and give yourself enough time for future assignments.

yaditya20 committed:
Added Landscape Layout and Polished the UI

yaditya20 committed:
Added Missing files from the last commit.

yaditya20 commented:
I have updated the existing layout and added the landscape layout and also I have resolved all the warnings in MainActivity.kt
Could you please review it if there's scope for improvement. And finally, if you can let me know if I can merge this PR to the master branch.
Thank you! 

Updating the task list that has been accomplished so far:

- [x] Portrait Layout of the app

- [x] Change Favorite Book on Click of a Button

- [x] Animation on Button Click and Book Change TextView

- [x] Landscape Layout of the app

- [x] Improve / Polish Layout

yaditya20 committed:
Fixed the Layout issue.

yaditya20 commented:
Merging week3 branch, in order to create a new PR for next week's assignment.
